elasticdl/pkg/ps/parameter.go

```diff
@@ -10,13 +10,17 @@ import (
 type Parameter struct {
 	NonEmbeddingParam map[string]*common.Tensor
 	EmbeddingParam    map[string]*common.EmbeddingTable
+	Version           int32
+	InitStatus        bool
 }
 
 // NewParameter creates a parameter instance
 func NewParameter() *Parameter {
 	var p Parameter
 	p.NonEmbeddingParam = make(map[string]*common.Tensor)
 	p.EmbeddingParam = make(map[string]*common.EmbeddingTable)
+	p.Version = 0
+	p.InitStatus = false
```

Comment:
How about let `NewParameter` to call make and assign to `NonEmbeddingParam` and `EmbeddingParam`.  The user could simply call

```go
p := NewParameter(0, false)
```

Reply:
It seems that the default value of int is 0, bool is false in Go.
Remove these two lines code.